cmd/ssh.go

```diff
@@ -472,6 +473,12 @@ func (cmd *SSHCmd) startProxyServices(
 		if gitCredentials {
 			command += " --configure-git-helper"
 		}
+
+		// check if we should enable git ssh commit signature support
+		if cmd.GitSSHSignatureForwarding || devPodConfig.ContextOption(config.ContextOptionGitSSHSignatureForwarding) == "true" {
```

Comment:
What happens if gpg agent forwarding is enabled as well?

Reply:
And by setting this we enable it for everyone even though they might not have set it up locally 🤔
Imo the intuitive way would be to enable it if it's available locally but opt-in if not. What do you think?

Reply:
yeah that's the idea, working on it atm. For testing so far it was just easier for me to have it enabled all the time. That's how I wanted to set it up:
- if you have ssh signing set up locally we'll automatically set it up for you 
- if you don't you'll be able to specify a flag with path to key you want to use for signing and we'll set it up 

And about the gpg agent forwarding - it wouldn't matter, as long as `gpg.format=ssh` is set git will use our tool. So gpg agent would be just waiting there but wouldn't be used. I'll do some testing though